go-lib/lassie-ffi.go

```diff
@@ -100,10 +100,6 @@ func InitDaemon(cfg *C.daemon_config_t) C.daemon_init_result_t {
 	// TODO: configure bitswap concurrency
 	// lassieOpts = append(lassieOpts, lassie.WithBitswapConcurrency(bitswapConcurrency))
 
-	if tempDir != "" {
-		lassieOpts = append(lassieOpts, lassie.WithTempDir(tempDir))
-	}
```

Comment:
doesn't this need changes in other parts too, if we can't pass a temp dir any more?

Reply:
I am glad you asked.

We are still providing temp-dir configuration to Lassie's HTTP sever.

https://github.com/filecoin-station/rusty-lassie/blob/9e9a40aa8da75c89687f3a606c91b320e4fe06c9/go-lib/lassie-ffi.go#L110-L117

Here is the Lassie PR removing `lassie.WithTempDir` API: https://github.com/filecoin-project/lassie/pull/290. I don't fully understand all ramifications of that change, but I believe it should not affect us.

Reply:
ok sounds good!